languages.toml

```diff
@@ -1813,7 +1813,7 @@ injection-regex = "sql"
 
 [[grammar]]
 name = "sql"
-source = { git = "https://github.com/DerekStride/tree-sitter-sql", rev = "25be0b8f17e9189ad9e1b875869d025c5aec1286" }
+source = { git = "https://github.com/DerekStride/tree-sitter-sql", rev = "4ffda1fe6841bcb2bfedb4a465c36ef8db0079b3" }
```

Comment:
tree-sitter-sql doesn't check in the generated files like `parser.c` into its main branch. You can use the latest revision from the `gh-pages` branch instead https://github.com/DerekStride/tree-sitter-sql/tree/gh-pages

Reply:
ah, right, I keep forgetting that